### PR TITLE
fixes for broken bwameth datamager 

### PR DESCRIPTION
**Pull request by bwlang**
before this patch the data manager fails with a key not found dict error

### CODE REVIEW

File: data_managers/data_manager_bwameth_index_builder/data_manager/bwameth_index_builder.py [hunk 1]
```diff
@@ -24,7 +24,7 @@ def build_bwameth_index(data_manager_dict, params, args):
     if return_code:
         print >> sys.stderr, "Error building index."
         sys.exit( return_code )
-    data_table_entry = dict(value=args.dbKey, dbkey=args.dbkey, name=args.name, path=args.dbKey)
+    data_table_entry = dict(value=args.dbkey, dbkey=args.dbkey, name=args.name, path=fasta_base_name)
```
Comment (mvdbeek) [inline data_managers/data_manager_bwameth_index_builder/data_manager/bwameth_index_builder.py]: I think this should be `sym_linked_fasta_filename`, which points to a symlink in the target directory, and I suppose that doubles as the basename for the bwa indices. Does that look right to you, @dpryan79 ?

Comment (dpryan79) [inline data_managers/data_manager_bwameth_index_builder/data_manager/bwameth_index_builder.py]: @mvdbeek I believe you're correct.

**Comment by bgruening** [comment]
ping @mvdbeek @dpryan79

**Comment by mvdbeek** [comment]
Excellent stuff, thanks everyone!

**Pull request merged**